src/ert/config/ert_script.py

```diff
@@ -143,7 +167,20 @@ def initializeAndRun(
             self._ert = fixtures.get("ert_config")
             self._ensemble = fixtures.get("ensemble")
             self._storage = fixtures.get("storage")
-            return self.run(*arguments)
+
+            positional_args = (
+                fixture_args
+                if is_using_wf_args_fixture
+                else [*workflow_args, *fixture_args]
+            )
+            if not positional_args and not use_kwargs:
```

Comment:
This we could keep this as it used to be? I dont think kwargs were really supported before, they just ended up being `workflow_args` in a list?